RELEASE-NOTES.md

```diff
@@ -1,5 +1,14 @@
 This file contains the RELEASE-NOTES of the Semantic Forms Select (a.k.a. SFS) extension.
 
+### 3.0.0
+
+not yet released
+
+* #53 Support "mapping property" / "mapping template" in value field (by Alexander Gesinn), requires changes in PF
+* #62 Make SFS compliant with recent PF versions (by Alexander Gesinn)
+* refactored SemanticFormsSelectInput class -> moved logic to new SelectField class (by Alexander Gesinn)
+* added Unit Tests (by Felix Ashu)
```

Comment:
Since the release notes are a guide for the users (wiki admins) in which they can see what the new versions bring and what they need to watch out for, things such as tests and refactoring are normally not mentioned. If a bug was caught and fixed by adding tests, then it should be mentioned.